common/test/acceptance/pages/lms/discussion.py

```diff
@@ -108,6 +108,10 @@ def has_add_response_button(self):
         """Returns true if the add response button is visible, false otherwise"""
         return self.is_element_visible(".add-response-btn")
 
+    def has_discussion_reply_editor(self):
+        """Returns true if the discussion reply editor is is visible"""
```

Comment:
According to Docstring guidelines: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings, this should be changed to:
```
"""
Returns true if the discussion reply editor is is visible
"""
```